src/DoubleType.java

```diff
@@ -0,0 +1,5 @@
+package src;
+
+public interface DoubleType{
```

Comment:
Je hebt een goede oplossing bedacht om dubbele types te implementeren. 

De implementatie kan nog wel iets beter, maar de aanzet is goed.
Ik zou verwachten dat je deze interface bij een klasse `WaterFirePokemon` implementeert, die `Firepokemon` of `WaterPokemon` extend en dan deze `DoubleType` implementeert, maar je hebt het bij `FirePokemon` geimplementeerd die de DoubleType als optionele functionaliteit heeft. Dat werkt net wat minder goed.